tests/unit/test_metadata.py

```diff
@@ -25,6 +25,15 @@ class DifferentColor(Enum):
     BROWN = 3
 
 
+class Countries(IntEnum):
+    """Countries enum used for testing enum-typed config and output."""
```

Comment:
Please call out that this tests `IntEnum`

Reply:
Updated comment as suggested.